client/src/pages/account/notifications/index.js

```diff
@@ -8,32 +8,25 @@ import { useUser } from 'hooks/useUser'
 import api from 'api'
 
 const Notifications = () => {
-  const { notifications } = useNotifications()
-  const [didUpdateNotifs, setdidUpdateNotifs] = React.useState(false)
   const { refreshUserData, user, token } = useUser()
+  const { getLastNotificationDate, fetchNotifications } = useNotifications()
+  const [didUpdateNotifs, setdidUpdateNotifs] = React.useState(false)
+  const [notifications, setNotifications] = React.useState([])
 
   React.useEffect(() => {
-    if (!didUpdateNotifs) {
+    const updateNotifsViewed = async () => {
+      await fetchNotifications().then((notifs) => {
```

Comment:
since the fetchNotifications is already async await can we have it `const notifs = fetchNotifications()` and then call `setNotifications`

Reply:
Cool. done